docs/VISION_DOCUMENT.md

```diff
@@ -0,0 +1,347 @@
+| Definição de atributos para a fila de espera | Alta | Padronizar e tornar mais eficiente o sistema de posicionamento da fila de espera | Classificação subjetiva de acordo com o diagnóstico feito pelo médico | Tornar os critérios para a posição na fila de espera objetivos e padronizados |
+
+### 3.7	Alternativas e Concorrência
+
+Atualmente não há nenhum aplicativo que integre pacientes de SD, familiares e profissionais - o que é proposto neste projeto.  Há um aplicativo específico para o acompanhamento à saúde das crianças com Síndrome de Down (DownEx) e outros aplicativos que atendem pessoas com deficiência intelectual, que poderiam ser utilizados por pessoas com Síndrome de Down. Contudo, como tais aplicativos têm foco pedagógico e educacional, não são considerados concorrentes.
+
+
+## 4:	Visão Geral do Produto
+
+### 4.1	Perspectiva do Produto
+
+ O sistema Dr. Down funciona em múltiplas camadas com a finalidade de atender as necessidades de seus usuários.  A função do sistema pode ser resumida em auxiliar a equipe do Cris Down no cadastro e gerenciamento de pacientes, o que inclui: marcação de consultas, fila de espera, além da divulgação de informações importantes. Para possibilitar a execução dos propósitos do sistema, ele deverá possuir acesso aos dados da Secretaria de Estado de Saúde (SES) através de um número de identificação, o que possibilitará levantar dados de determinados pacientes.
+
+### 4.2	Resumo das Capacidades
+
+|Benefícios para o Cliente |Recursos de |
+|------------------|-----------------|
+| Consulta rápida do histórico do paciente |Pesquisa do banco de dados do Cris Down. |
+| Comunicação entre pacientes, familiares e médicos |Fórum para dúvidas e mensagens diretas. |
+| Verificação do paciente na fila de espera | Funcionalidade mostrando o tempo que falta até a sua consulta. |
+| Jogos para o desenvolvimento dos pacientes | Ligação da página com links para jogos interativos |
+
+### 4.3	Licenciamento e Instalação
+
+A distribuição do software se dará sob a licença do MIT. Tal licença é aberta quanto a permissão para edição, visualização e utilização do software.
+
+## 5:	Recursos do Produto
+
+### 5.1 Acesso.
+
+A autenticação do usuário deverá ocorrer por login.
+
+### 5.2 Nível de acesso
+
+As informações dependem do tipo de usuário. Os pacientes e familiares terão acesso apenas às informações básicas do prontuário. Caso seja um funcionário do hospital, possuirá, além das informações básicas do paciente, o número de SES e poderá inserir informações sobre o atendimento realizado por ele. O médico, por fim, terá acesso a todas as funcionalidades básicas, além de modificar/atualizar informações no sistema sobre o paciente.
+
+### 5.3 Cadastro
+
+O cadastro ocorrerá no próprio aplicativo. Quando o perfil for de um  funcionário do hospital ou médico do Cris Down, dependerá de um administrador verificar as informações dos profissionais da rede.
```

Comment:
- "O cadastro ocorrerá no próprio aplicativo. Quando o perfil for de um  funcionário do hospital ou médico do Cris Down ~~, dependerá de um administrador~~ *. Dependerá de um administrador do sistema* verificar as informações dos profissionais da rede."